templates/components/logs.html.twig

```diff
@@ -123,7 +123,7 @@
                   <td>{{ entry['date_mod']|formatted_datetime }}</td>
                   <td>{{ entry['user_name'] }}</td>
                   <td>{{ entry['field'] }}</td>
-                  <td colspan="2" style="width: 60%">{{ entry['change']|raw }}</td>
+                  <td colspan="2" style="width: 60%">{{ entry['change']|safe_html }}</td>
```

Comment:
I think we should not rely on `|safe_html` here to fix the encoding. `entry['change']` is built at runtime by GLPI. The encoding should be done in the method that inserts values between `<ins>` and `<del>` HTML tags.

Reply:
Since the method return an array of values and does not really output anything, no; it should not escape. I have in mind that is the rule.

Reply:
The method itself returns some HTML code (`<ins>` and `<del>`), so this HTML code should be "safe". Also, if there are special chars in the old /new value, they should be displayed.

![image](https://github.com/user-attachments/assets/0e4acd9a-03fa-44e8-99cd-8d79a70cbde4)